test/jdk/javax/swing/JOptionPane/8081019/bug8081019.java

```diff
@@ -57,8 +56,7 @@ public static void main(String[] args) throws Exception {
         }
     }
 
-    private static void runTest() throws Exception {
```

Comment:
guess command is always "RUN_PROCESS" AFAICS so is there any need to keep this `runTest`...I dont see any @run in test to test both RUN_PROCESS and RUN_TEST so probably test can be simplified or add 2 run tags to test both or am I missing something?